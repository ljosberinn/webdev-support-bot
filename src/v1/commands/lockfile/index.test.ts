import handleCodeRequest from '.';

test('shows how to reset lockfile', async () => {
  const sendMock = jest.fn();
  const msg: any = {
    channel: { send: sendMock },
  };

  await handleCodeRequest(msg);
  const response = msg.channel.send.mock.calls[0][0];
  expect(response).toMatchSnapshot();
});
