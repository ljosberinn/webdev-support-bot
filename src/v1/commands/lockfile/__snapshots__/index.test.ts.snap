// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows how to reset lockfile 1`] = `
"**How to reset your lockfile:**

1. Remove your lockfile â€” it is either \`package-lock.json\` or when using yarn \`yarn.lock\`
2. Delete the node_modules directory
3. Install the dependencies again with either \`npm install\` or \`yarn\`
"
`;
